### PR TITLE
Use editorial selection in homepage for dataset and reuse

### PR DESCRIPTION
**Pull request by noirbizarre**
No more automatic selection in homepage, it now use the manual editorial selection.

The default home template alos use real data now.

### CODE REVIEW

File: udata/templates/home.html [hunk 1]
```diff
@@ -36,81 +37,77 @@ <h1 class="page-header">Titre de l'actu
...
-                                <a href="#">
-                                    <img class="img-responsive portfolio-item" src="http://placehold.it/300x200">
-                                </a>
-                            </div>
-
-                            <div class="col-sm-3 col-xs-6">
-                                <a href="#">
-                                    <img class="img-responsive portfolio-item" src="http://placehold.it/300x200">
-                                </a>
+                            {% for dataset in reuse.datasets[:4] %}
```
Comment (davidbgk) [inline udata/templates/home.html]: I think that it's more "secure" to put limits in the view but YMMV.

**Comment by davidbgk** [comment]
Do we remove the related code in udata-gouvfr in the meantime?

**Comment by noirbizarre** [comment]
I already removed some parts (etalab/udata-gouvfr#16).

More will require a theme refactoring, which, I think, will be an whole story by itself.

File: udata/templates/home.html (hunk 1 above)
Comment (noirbizarre) [inline udata/templates/home.html]: Yep, I was wondering which one should I choose between
- no limits in view and slicing in template => allow theme to limit itself the amount of required datasets/reuses
- limits in view => theme is limited by the view itself

Comment (davidbgk) [inline udata/templates/home.html]: Oh good point. Let's discuss that tomorrow.

Comment (noirbizarre) [inline udata/templates/home.html]: Good to me.

**Comment by davidbgk** [comment]
r+

**Pull request merged**